settings.gradle

```diff
@@ -24,8 +24,8 @@ include "test-suite"
 include "test-suite-geb"
 include "test-suite-keycloak"
 include "test-suite-keycloak-16"
-include "test-suite-tomcat"
-include "test-suite-tomcat-groovy"
+//include "test-suite-tomcat"
```

Comment:
I have temporary comment out this test modules until servlet is ready for 4.0.0-SNAPSHOT